Source/WebCore/testing/Internals.idl

```diff
@@ -815,7 +815,7 @@ enum RenderingMode {
     DOMString pageProperty(DOMString propertyName, long pageNumber);
     DOMString pageSizeAndMarginsInPixels(long pageIndex, long width, long height, long marginTop, long marginRight, long marginBottom, long marginLeft);
 
-    undefined setPageScaleFactor(unrestricted float scaleFactor, long x, long y);
+    Promise<undefined> setPageScaleFactor(unrestricted float scaleFactor, long x, long y);
```

Comment:
Have we considered moving this to TestRunner.idl? Going to the UIProcess is exactly what testRunner API does. Whereas internals.idl API is usually expected to set some state directly inside the WebProcess.